mergify_engine/worker.py

```diff
@@ -907,6 +907,24 @@ def get_process_index_from_env() -> int:
         return 0
 
 
+async def ping_redis(
+    redis: typing.Union[redis_utils.RedisStream, redis_utils.RedisCache],
+    redis_name: str,
+) -> None:
+    def retry_log(retry_state: tenacity.RetryCallState) -> None:
+        statsd.increment("redis.client.connection.errors")
+        LOG.warning(
+            f"Couldn't connect to Redis {redis_name}, retrying...",
```

Comment:
if we can tell in how many time we will retry :)
```suggestion
            f"Couldn't connect to Redis {redis_name}, retrying in X seconds...",
```